Xamarin.Forms.Platform.Android/Renderers/ShellFlyoutTemplatedContentRenderer.cs

```diff
@@ -62,7 +63,7 @@ protected virtual void LoadView(IShellContext shellContext)
             var adapter = new ShellFlyoutRecyclerAdapter(shellContext, OnElementSelected);
             recycler.SetPadding(0, (int)context.ToPixels(20), 0, 0);
             recycler.SetClipToPadding(false);
-            recycler.SetLayoutManager(new LinearLayoutManager(context, (int)Orientation.Vertical, false));
+			recycler.SetLayoutManager(_layoutManager = new ScrollLayoutManager(context, (int)Orientation.Vertical, false) { ScrollVertically = false });
```

Comment:
I feel like `ScrollVertically` should be set to `_shellContext.Shell.FlyoutVerticalScroll`. Is that not correct?

Reply:
good catch 👍